src/main/java/com/github/forax/umldoc/Main.java

```diff
@@ -33,35 +33,43 @@ public static void main(String[] args) {
       return;
     }
     var finder = ModuleFinder.of(Path.of(args[0]));
-    var module = finder.find("com.github.forax.umldoc");
+    var modules = finder.findAll().iterator();
 
-    if (module.isEmpty()) {
-      System.err.println("Couldn't find the Module");
+    if (!modules.hasNext()) {
+      System.err.println("Couldn't find a Module");
       return;
     }
     List<Package> packages;
     try {
-      packages = ModuleScrapper.scrapModule(module.get());
+      packages = ModuleScrapper.scrapModule(modules.next());
     } catch (IOException e) {
       System.err.println("Couldn't get the list of Packages : " + e.getMessage());
       return;
```

Comment:
call System.exit() with a non zero code

Reply:
According to the javadoc, -1 is the value to use when the program is not successful.

Should the exit method take as a parameter : -1 or another value ?

Reply:
-1 is great